r1cs/gadgets/crypto/src/crh/poseidon/tweedle/mod.rs

```diff
@@ -20,3 +20,6 @@ pub type TweedleFrPoseidonHashGadget = PoseidonHashGadget<
     TweedleFrQuinticSbox,
     TweedleFrQuinticSboxGadget,
 >;
+
+pub mod density_optimized;
```

Comment:
As pointed out above, let us "change the role" of density-optimized vs. non density-optimized. For example, the standard case is just something like mod Poseidon256, and the generic not-optimized implementation will be Poseidon_non_optimized.

Reply:
Let's address this in another PR in which we will generalize this design.